src/Resources/config/services.yml

```diff
@@ -2,13 +2,19 @@ services:
     ambta_doctrine_encrypt.subscriber:
         alias: ambta_doctrine_encrypt.orm_subscriber
 
+    ambta_doctrine_encrypt.encrypt_service:
+        class: Ambta\DoctrineEncryptBundle\Service\Encrypt
+        arguments: ["@ambta_doctrine_encrypt.encryptor"]
+        public: true
```

Comment:
I would not make the service public, I don't see the need to be able to be able to run `$container->get('ambta_doctrine_encrypt.encrypt_service')`.

Making services public is bad practice since symfony 4.x, when looking at the [symfony documentation](https://symfony.com/doc/6.4/service_container.html#public-versus-private-services): 

> As a best practice, you should only create private services and you should fetch services using dependency injection instead of using `$container->get()`.